### ISSUE TITLE
MySQL full-copy - ddl - table partitioning - partition by list cause src task error

### ISSUE THREAD

**Issue by asiroliu** [actiontech/dtle#803]
## Description
MySQL full-copy - ddl - table partitioning - partition by list cause src task error

## Steps to reproduce the issue
1.  create table
```sql
SQL> CREATE TABLE t7 (id INT, name VARCHAR(35)) PARTITION BY LIST (id) (PARTITION r0 VALUES IN (1, 5, 9, 13, 17, 21), PARTITION r1 VALUES IN (2, 6, 10, 14, 18, 22), PARTITION r2 VALUES IN (3, 7, 11, 15, 19, 23), PARTITION r3 VALUES IN (4, 8, 12, 16, 20, 24));
```
2. create dtle job

## Describe the results you received
src dtle log
```
2021-11-09T13:18:08.650+0800 [WARN]  client.driver_mgr.dtle: No valid unique key found. It will be slow on large table.: driver=dtle @module=inspector job=aaa-migration nKey=0 schema=test table=t7 timestamp=2021-11-09T13:18:08.650+0800
2021-11-09T13:18:08.652+0800 [ERROR] client.driver_mgr.dtle: error at ParseCreateTableStmt.: driver=dtle @module=dtle.extractor err="line 9 column 62 near "LIST (id)
(PARTITION r0 VALUES IN (1,5,9,13,17,21) ENGINE = InnoDB,
 PARTITION r1 VALUES IN (2,6,10,14,18,22) ENGINE = InnoDB,
 PARTITION r2 VALUES IN (3,7,11,15,19,23) ENGINE = InnoDB,
 PARTITION r3 VALUES IN (4,8,12,16,20,24) ENGINE = InnoDB) */" " job=aaa-migration timestamp=2021-11-09T13:18:08.651+0800
2021-11-09T13:18:08.652+0800 [INFO]  client.driver_mgr.dtle: Step: committing transaction: driver=dtle job=aaa-migration n=2 @module=dtle.extractor timestamp=2021-11-09T13:18:08.651+0800
2021-11-09T13:18:08.652+0800 [ERROR] client.driver_mgr.dtle: onError: driver=dtle @module=dtle.extractor err="line 9 column 62 near "LIST (id)
(PARTITION r0 VALUES IN (1,5,9,13,17,21) ENGINE = InnoDB,
 PARTITION r1 VALUES IN (2,6,10,14,18,22) ENGINE = InnoDB,
 PARTITION r2 VALUES IN (3,7,11,15,19,23) ENGINE = InnoDB,
 PARTITION r3 VALUES IN (4,8,12,16,20,24) ENGINE = InnoDB) */" " job=aaa-migration timestamp=2021-11-09T13:18:08.652+0800
2021-11-09T13:18:08.652+0800 [DEBUG] client.driver_mgr.dtle: extractor shutdown: driver=dtle @module=dtle.extractor job=aaa-migration timestamp=2021-11-09T13:18:08.652+0800
2021-11-09T13:18:08.652+0800 [INFO]  client.driver_mgr.dtle: extractor shutdown: driver=dtle @module=dtle.extractor job=aaa-migration timestamp=2021-11-09T13:18:08.652+0800
2021-11-09T13:18:08.652+0800 [INFO]  client.driver_mgr.dtle: Shutting down: driver=dtle job=aaa-migration @module=dtle.extractor timestamp=2021-11-09T13:18:08.652+0800
2021-11-09T13:18:08.653+0800 [INFO]  client.driver_mgr.dtle: DestroyTask: driver=dtle @module=dtle id=9cc36f8d-002f-329b-55a0-12a266b00895/src/aaf642c2 timestamp=2021-11-09T13:18:08.653+0800
2021-11-09T13:18:08.653+0800 [DEBUG] client.driver_mgr.dtle: extractor shutdown: driver=dtle @module=dtle.extractor job=aaa-migration timestamp=2021-11-09T13:18:08.653+0800
```

## Describe the results you expected
the table should copy to the dest

## Output of `./dtle version`:**

```
9.9.9.9-master-89033ce
```

## Additional information
src mysql show
```sql
mysql> show create table t7\G
*************************** 1. row ***************************
       Table: t7
Create Table: CREATE TABLE `t7` (
  `id` int(11) DEFAULT NULL,
  `name` varchar(35) DEFAULT NULL
) ENGINE=InnoDB DEFAULT CHARSET=latin1
/*!50100 PARTITION BY LIST (id)
(PARTITION r0 VALUES IN (1,5,9,13,17,21) ENGINE = InnoDB,
 PARTITION r1 VALUES IN (2,6,10,14,18,22) ENGINE = InnoDB,
 PARTITION r2 VALUES IN (3,7,11,15,19,23) ENGINE = InnoDB,
 PARTITION r3 VALUES IN (4,8,12,16,20,24) ENGINE = InnoDB) */
1 row in set (0.00 sec)
```

**Comment by ffffwh** [comment]
fixed after 232ff9ce.

**Comment by asiroliu** [comment]
verified version:
9.9.9.9-master-aa4de8d
